tools/FhirSchemaManager/Resources.resx

```diff
@@ -132,12 +132,24 @@
   <data name="CurrentCommandDescription" xml:space="preserve">
     <value>Returns the current SQL schema version(s) from the given FHIR server.</value>
   </data>
+  <data name="CurrentDefaultErrorDescription" xml:space="preserve">
+    <value>Current version information is not available.</value>
+  </data>
+  <data name="CurrentInformationNotFound" xml:space="preserve">
+    <value>Current version information is not found.</value>
+  </data>
   <data name="FhirServerOptionDescription" xml:space="preserve">
     <value>The root URL of the FHIR server to connect to.</value>
   </data>
   <data name="FhirServerRequiredValidation" xml:space="preserve">
     <value>You must include a FHIR server.</value>
   </data>
+  <data name="HttpRequestExceptionMessage" xml:space="preserve">
+    <value>No connection could be made to the target machine.</value>
+  </data>
+  <data name="RequestFailedMessage" xml:space="preserve">
+    <value>Request has failed due to fhirServerUrl is invalid or inactive.</value>
```

Comment:
nit: probably need a placeholder for `fhirServerUrl` since it's a parameter name. 

Do we know when the server is invalid vs when it's unreachable?

Reply:
Right, when the url is invalid, exception is thrown by validators itself. So now updated errorMessage just for inactive.